docs/concepts/why-loopscale.md

```
@@ -0,0 +1,80 @@
+---
+sidebar_position: 2
+---
+
+# Why Loopscale?
+
+Before exploring Loopscale's new primitives and advantages, let's understand how traditional, pool-based lending protocols work and their limitations.
+
+## The problem with pool-based lending
+In the first generation of DeFi lending protocols, the borrowing interest rate is determined by the utilization of the asset. This model is known as a pool- or utilization-based model. As illustrated in the chart below, when utilization increases, borrowing is discouraged and supplying is encouraged. This design decision was made to maintain excess liquidity for withdrawals.
+
+![](../../static/img/docs/concepts/why-loopscale-1.png)
```

markdownlint-cli2 (0.17.2)

Images should have alternate text (alt text)
null

(MD045, no-alt-text)